src/libmongoc/tests/test-mongoc-topology.c

```diff
@@ -2430,13 +2430,13 @@ _test_hello_ok (bool pooled)
 }
 
 static void
-test_hello_ok_single ()
+test_hello_ok_single (void)
 {
    _test_hello_ok (false);
```

Comment:
Can you fix the `maybe-uninitialized` warnings in the `_test_hello_ok` function? The abi-compliance-check task ([example]( https://spruce.mongodb.com/task/mongo_c_driver_releng_abi_compliance_check_patch_bebe063a42c87cabd8b628e61e9901da1161a1a7_619430bf56234348fc2946dc_21_11_16_22_29_21/logs?execution=1)) with gcc 7.5.0 appears to fail from those.